composer.json

```diff
@@ -26,7 +26,7 @@
         "zendframework/zend-servicemanager": "^2.7 || ^3.0",
         "zendframework/zend-stdlib": "^2.2 || ^3.0",
         "slm/queue": "^1.0",
-        "doctrine/doctrine-orm-module":  "~0.7 || ^1.0 || ^2.1",
+        "dasprid/container-interop-doctrine": "^1.1",
         "doctrine/dbal": "^2.5"
     },
     "require-dev": {
```

Comment:
perhaps we can use the "suggest" key of composer to suggest using `doctrine/doctrine-orm-module`

Reply:
`"doctrine/doctrine-orm-module": "if you want use Doctrine ORM Module"` is ok? Or we should set another description?

Reply:
Perhaps, ... 

"doctrine/doctrine-orm-module" : "^2.0 to use DoctrineORMModule as dasprid/container-interop-doctrine implementation"

Reply:
Done